sipa/config/default.py

```diff
@@ -114,3 +114,7 @@
 
 # link to clandar
 MEETINGS_ICAL_URL = "https://agdsn.de/cloud/remote.php/dav/public-calendars/bgiQmBstmfzRdMeH?export"
+
+# statuspage
+STATUS_PAGE_ENDPOINT = "https://status.agdsn.net/api/subscribers/subscribers/"
+STATUS_PAGE_AUTH = ""
```

Comment:
trailing newline missing

Reply:
PyCharm somehow swallows the line break at the end of the file